python/cugraph/cugraph/layout/force_atlas2.py

```diff
@@ -23,15 +64,20 @@ def force_atlas2(
     outbound_attraction_distribution=True,
     lin_log_mode=False,
     prevent_overlapping=False,
+    vertex_radius=None,
+    overlap_scaling_ratio=100.0,
     edge_weight_influence=1.0,
     jitter_tolerance=1.0,
     barnes_hut_optimize=True,
     barnes_hut_theta=0.5,
     scaling_ratio=2.0,
     strong_gravity_mode=False,
     gravity=1.0,
+    mobility=None,
```

Comment:
Why are the new parameters `vertex_radius` and `mobility`? Would e.g. `vertex_mobility` be more clear and consistent?

Reply:
> Why are the new parameters `vertex_radius` and `mobility`? Would e.g. `vertex_mobility` be more clear and consistent?

Sounds good to me!

Reply:
@eriknw shall I `/merge` or do you want to make this change?

Reply:
I agree that it seems more consistent so I'm in favor.
I'll add a `DO_NOT_MERGE` since this has been approved so it doesn't get merged by someone else before you get that change in.

Reply:
I pushed the change and removed the `DO_NOT_MERGE` label